assignments/objects.js

```diff
@@ -18,27 +18,66 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  "id": 0,
+  "name": "Mitzi",
+  "email":"mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const kennan = {
+  "id": 1,
+  "name": "Kennan",
+  "email":"kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function (){
+    console.log('Hello, my name is Kennan!')
+  }
+}
+const keven = {
+  "id": 2,
+  "name": "Keven",
+  "email":"kmummery2@wikimedia.org",
+  "gender": "M"
+}
+const gannie = {
+  "id": 3,
+  "name": "Gannie",
+  "email":"gmartinson3@illinois.edu",
+  "gender": "M"
+}
+const antonietta = {
+  "id": 4,
+  "name": "Antonietta",
+  "email":"adaine5@samsung.com",
+  "gender": "F",
+  multiplyNums: function (){
+    console.log(3*4)
+  }
+}
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(mitzi.name);
 
 // Kennan's ID
-
+console.log(kennan.id);
 // Keven's email
-
+console.log(keven.email);
 // Gannie's name
-
+console.log(gannie.name);
 // Antonietta's Gender
+console.log(antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(kennan.speak());
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+console.log(antonietta.multiplyNums(3,4));
```

Comment:
This is the solution: 
antonietta.multiplyNums = function(a,b) {
  return a * b;
}
console.log(antonietta.multiplyNums(3,4));
The reason your code works is bc you set the numbers on the object, but it's now limited to only multiplying those specific numbers